### ISSUE TITLE
Synchronization with other services?

### ISSUE THREAD

**Issue by jebeld17** [anthonycr/Lightning-Browser#476]
I don't know if this is possible, but I would love to have an ability for Lightning Browser to synchronize with external services such as Microsoft, Google, Firefox, or iCloud to synchronize my browsing data and bookmarks.

**Comment by kuc** [comment]
I don't think that this is in scope of this project, even if e.g. Microsoft's employee would have come here and made ready-to-merge pull request.
